src/main/java/com/google/sps/servlets/actions/ProjectsServlet.java

```diff
@@ -40,8 +40,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         String jsonResult = UrlRequest.urlQuery(API_PATH, queryParams);
         List<Project> projectsList = extractProjectsList(jsonResult);
         jsonResultMap.put(term, projectsList);
-      } catch(NullPointerException npe) {
-        npe.printStackTrace();
+      } catch(Exception e) {
```

Comment:
Why was this broadened to catch any exception?  Are there multiple types of exceptions that can occur now?

Reply:
Yeah, when there are no results there's a NullPointerException but when there's a problem with the API_KEY or with the API itself there's another one.

Reply:
See comment above.